### PR TITLE
Create new govuk wrapper and templates

### PR DESCRIPTION
**Pull request by llmikeyj**

### CODE REVIEW

File: app/views/templates/agent_main_template.scala.html [hunk 1]
```diff
@@ -63,7 +63,7 @@
     }
 
 @headStyle = {
-        <link rel="stylesheet" href="@routes.Assets.at("stylesheets/banner-panel.css")">
+        <link rel="stylesheet" href="@routes.Assets.versioned("stylesheets/banner-panel.css")">
 }
 
 @prefixedTitle = @{
```
Comment (abiodxn) [inline app/views/templates/agent_main_template.scala.html:69]: Do we still need this? We have AgentMainTemplate.scala ?

File: app/views/templates/injected/MainTemplate.scala.html [hunk 2]
```diff
@@ -48,16 +48,16 @@
...
         @* append custom scripts *@
 @scriptElem.fold(HtmlFormat.empty)(x => x)
 }
 
 @headStyle = {
-        <link rel="stylesheet" href="@routes.Assets.at("stylesheets/banner-panel.css")">
+        <link rel="stylesheet" href="@routes.Assets.versioned("stylesheets/banner-panel.css")">
 }
 
 @prefixedTitle = @{
```
Comment (abiodxn) [inline app/views/templates/injected/MainTemplate.scala.html:63]: Is this used anywhere? We have main template, and principal main template ?

File: app/views/templates/main_template.scala.html [hunk 3]
```diff
@@ -64,7 +64,7 @@
 }
 
 @headStyle = {
-        <link rel="stylesheet" href="@routes.Assets.at("stylesheets/banner-panel.css")">
+        <link rel="stylesheet" href="@routes.Assets.versioned("stylesheets/banner-panel.css")">
 }
 
 @prefixedTitle = @{
```
Comment (abiodxn) [inline app/views/templates/main_template.scala.html:70]: Do we need both main_template and MainTemplate ?

File: app/views/templates/agent_main_template.scala.html (hunk 1 above)
Comment (llmikeyj) [reply app/views/templates/agent_main_template.scala.html:69]: I think we do; this ticket is not about decommissioning the old templates, and the changes to the generated ReverseRoutes are breaking changes

File: app/views/templates/injected/MainTemplate.scala.html (hunk 2 above)
Comment (llmikeyj) [reply app/views/templates/injected/MainTemplate.scala.html:63]: This appears to have been left over from some previous experiment. Advice from Alex was to leave it for now, in anticipation of a proper clean-up once we've migrated over to the new design system.

File: app/views/templates/main_template.scala.html (hunk 3 above)
Comment (llmikeyj) [reply app/views/templates/main_template.scala.html:70]: I think we do, as per previous comments

**Pull request merged**